google/ggrc-core
# GGRC-3982 Remove view-only columns from csv templates

# Issue description

Remove view-only fields from csv-templates

# Steps to test the changes

Steps to reproduce:
1. Go to Import page
2. Download Assessment csv template
3. Open csv template
*Actual Result:* 'Last Deprecated Date' is displayed in Assessment import spreadsheet
*Expected Result:* 'Last Deprecated Date'  should not be displayed in Assessment import spreadsheet


# Solution description
Do not include columns that have view_only attribute if only headers are outputted (object_ids list is empty)

# Sanity checklist




# PR Review checklist
